docs/sources/flow/reference/components/discovery.kubelet.md

```diff
@@ -0,0 +1,145 @@
+---
+title: discovery.kubelet
```

Comment:
I would like to label this has BETA until we get some more confidence in using it.  Add 
```
labels:
  stage: beta
```
under the title